ros2param/ros2param/api/__init__.py

```diff
@@ -14,112 +14,38 @@
 
 import sys
 
-from rcl_interfaces.msg import Parameter
-from rcl_interfaces.msg import ParameterType
-from rcl_interfaces.msg import ParameterValue
-from rcl_interfaces.srv import DescribeParameters
-from rcl_interfaces.srv import GetParameters
-from rcl_interfaces.srv import ListParameters
-from rcl_interfaces.srv import SetParameters
 import rclpy
-from rclpy.parameter import PARAMETER_SEPARATOR_STRING
+from rcl_interfaces.msg import ParameterType
+from rclpy.parameter import parameter_value_to_python
+from rclpy.parameter import get_parameter_value as rclpy_get_parameter_value
+from rclpy.parameter_client import AsyncParameterClient
+
 from ros2cli.node.direct import DirectNode
 
-import yaml
+
+def get_parameter_value(*, string_value):
```

Comment:
If we've replaced all calls to this function in ros2cli, then I guess we add a deprecation warning here. You can `import warnings` and then call `warnings.warn('get_parameter_value() is deprecated. Use rclpy.parameter.get_parameter_value instead')` in this function.